templates/registration/login.html

```diff
@@ -3,22 +3,23 @@
 
 {% block content %}
 
-{% if form.errors %}
-  <p>Your username and password didn't match. Please try again.</p>
-{% endif %}
+  {% if form.errors %}
+    <p>Your username and password didn't match. Please try again.</p>
+  {% endif %}
 
-<h1>Login</h1>
+  <h1>Login</h1>
 
-{% if next %}
-  <p>Please login to see this page.</p>
-{% endif %}
+  {% if next %}
+    <p>Please login to see this page.</p>
+  {% endif %}
 
-<form method="post" action="{% url 'login' %}">
-  {% csrf_token %}
-  {{ form|crispy }}
+  <form method="post" action="{% url 'login' %}">
```

Comment:
Ensure that the URL name 'login' used in the form action matches the URL configuration in your Django project. If the URL pattern is named differently, update this to prevent form submission errors.